src/frontend/js/components/SearchFiltersPane/index.tsx

```diff
@@ -52,10 +52,11 @@ export const SearchFiltersPane = ({
       <h2 className="search-filters-pane__title">
         <FormattedMessage {...messages.filter} />
       </h2>
-      <a
+      <span
```

Comment:
Maybe this could just be a `button` with styles that make it look plain then?

(unlike pagination I agree this one might be better as a button)